Lib/lib2to3/tests/test_parser.py

```diff
@@ -61,6 +61,9 @@ def test_load_grammar_from_pickle(self):
             shutil.rmtree(tmpdir)
 
     @unittest.skipIf(sys.executable is None, 'sys.executable required')
+    @unittest.skipIf(
+        sys.platform == 'emscripten', 'requires working subprocess'
```

Comment:
why not @requires_subprocess ?

Reply:
Because I made a mistake and forgot to change the code :)

Reply:
Ah no! Now I remember. The lib2to3 package has its own test suite that does not use anything from ``test.support``.

Reply:
yuck, but it's a doomed library so no reason to change that now.